frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -157,7 +157,7 @@ function BookInfo:show(file, book_props)
     UIManager:show(widget)
 end
 
-function BookInfo:getBookProps(file, book_props)
+function BookInfo:getBookProps(file, book_props, do_not_open)
     if DocSettings:hasSidecarFile(file) then
         local doc_settings = DocSettings:open(file)
```

Comment:
May be `no_open_document` (as here, you do `DocSettings:open()` even if one provides `do_not_open` ! :)